pom.xml

```diff
@@ -374,6 +374,18 @@
                             <fromFile>transaction-category-collector-client-api-v2.0.0.yaml</fromFile>
                         </configuration>
                     </execution>
+
+                    <execution>
+                        <id>download-positive-pay-client-api</id>
+                        <goals>
+                            <goal>download-single</goal>
+                        </goals>
+                        <phase>generate-sources</phase>
+                        <configuration>
+                            <url>https://artifacts.backbase.com/specs/positive-pay</url>
```

Comment:
BB-Fuel can't have references to artifacts. Only deliverables that area already in repo.